.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+const { getBaseConfig } = require("@edx/frontend-build");
+
+const config = getBaseConfig("eslint");
+
+config.rules = {
+  "import/no-extraneous-dependencies": [
```

Comment:
I'm assuming this stuff makes it the same as `frontend-app-learner-portal-enterprise`? 
I wonder if we should use this file as a joint `.eslintrc` for all our enterprise repos so they are all on the same version of everything....

Reply:
interesting thought!